FinSearchUnified/Helper/StaticHelper.php

```diff
@@ -204,7 +204,12 @@ public static function useShopSearch()
     {
         $request = Shopware()->Front()->Request();
         $isCLIMode = $request === null;
-        if ($isCLIMode) {
+
+        /*
+         * Shop is not available in the Backend for the search of the Product Stream preview.
```

Comment:
Sorry that I have to add a review issue for that, but we decided a long time ago on how comments should be written.
`//` is the way to go. E.g.

```php
// Bla bla bla, this is really important information,
// removing the following line will destroy the whole universe!
```